solidity/contracts/converter/types/fixed-rate-pool/FixedRatePoolConverter.sol

```diff
@@ -0,0 +1,315 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "../standard-pool/StandardPoolConverter.sol";
+
+/**
+ * @dev This contract is a specialized version of the converter, which is optimized for a
+ * liquidity pool that has 2 reserves with 50%/50% weights, and a conversion-rate of 1:1.
+ */
+contract FixedRatePoolConverter is StandardPoolConverter {
+    uint128 private _rateN;
+    uint128 private _rateD;
+
+    /**
+     * @dev initializes a new FixedRatePoolConverter instance
+     *
+     * @param  _anchor             anchor governed by the converter
+     * @param  _registry           address of a contract registry contract
+     * @param  _maxConversionFee   maximum conversion fee, represented in ppm
+     */
+    constructor(
+        IConverterAnchor _anchor,
+        IContractRegistry _registry,
+        uint32 _maxConversionFee
+    ) public StandardPoolConverter(_anchor, _registry, _maxConversionFee) {
+        _rateN = 1;
+        _rateD = 1;
+    }
+
+    /**
+     * @dev returns the converter type
+     *
+     * @return see the converter types in the the main contract doc
+     */
+    function converterType() public pure override returns (uint16) {
+        return 4;
+    }
+
+    /**
+     * @dev returns the rate
```

Comment:
specify the direction here ("1 unit of the first reserve token in...")

Reply:
Fixed.